api/libproject/serviceaccounts.rb

```diff
@@ -0,0 +1,27 @@
+require_relative "../../libproject/utils/common"
+
+class ServiceAccounts
+
+  CREDENTIALS_BUCKET_NAME = "all-of-us-workbench-test-credentials"
+
+  SA_FILE_NAMES = {
+    :directory_service => {
+      :dev => "fake-research-aou-5db00a85f01b.json",
+      # TODO(dmohs): production
+      :prod => nil,
+    }
+  }
+
+  def maybe_download(env_key, service_account_key)
+    common = Common.new
+
+    saved_sa_file_name = "#{service_account_key.to_s.gsub("_", "-")}-sa.json"
```

Comment:
Is this what produces directory-service-sa.json? Can we just use that string to keep it clearer?